src/libraries/System.Text.Json/Common/JsonSourceGenerationOptionsAttribute.cs

```diff
@@ -43,11 +43,16 @@ class JsonSerializerOptionsAttribute : JsonAttribute
         /// <summary>
         /// Specifies a built-in naming polices to convert JSON property names with.
         /// </summary>
-        public JsonKnownNamingPolicy NamingPolicy { get; set; }
+        public JsonKnownNamingPolicy PropertyNamingPolicy { get; set; }
 
         /// <summary>
         /// Specifies whether JSON output should be pretty-printed.
         /// </summary>
         public bool WriteIndented { get; set; }
+
+        /// <summary>
+        /// Specifies the source generation mode.
```

Comment:
Do we want to document how this GenerationMode interacts with JsonSerializableAttribute.GenerationMode?

Maybe something like:

```
        /// <summary>
        /// Specifies the source generation mode for types that don't explicitly set the mode with JsonSerializableAttribute.
        /// </summary>
```

Reply:
👍 thanks